x/emissions/module/rewards/scores.go

```diff
@@ -483,3 +531,50 @@ func EnsureAllWorkersPresentWithheld(
 
 	return values
 }
+
+// CalculateTopicInitialEmaScore calculates the initial EMA score for new participants in the next epoch
+// using the current active set statistics. The initial score is set below the active set threshold
+// by using the formula: lowestEmaScore - lambda * standardDeviationOfEmaScores
+func CalculateTopicInitialEmaScore(
+	ctx context.Context,
+	keeper keeper.Keeper,
+	activeScores []types.Score,
+) (alloraMath.Dec, error) {
+	// If there are no scores, return zero
+	if len(activeScores) == 0 {
+		return alloraMath.ZeroDec(), nil
+	}
+
+	// Get lambda parameter from module params
+	params, err := keeper.GetParams(ctx)
+	if err != nil {
+		return alloraMath.Dec{}, errors.Wrapf(err, "error getting module params")
+	}
+	lambda := params.LambdaInitialScore
+
+	// Calculate standard deviation of EMA scores in active set
```

Comment:
Calc `min` then standard dev

Reply:
resolving in https://github.com/allora-network/allora-chain/pull/712 due to ease